includes/class-wc-checkout.php

```diff
@@ -879,6 +879,12 @@ protected function process_customer( $data ) {
 				$customer->set_last_name( $data['billing_last_name'] );
 			}
 
+			$current_user = get_user_by( 'id', $customer_id );
```

Comment:
Since you now have display_name in the CRUD object, you can use the getter here instead of loading the user from WordPress.